src/Client/StatusWatcherClient/ContactsKeeper.php

```diff
@@ -242,6 +242,7 @@ private function checkLimitsExceeded(ImportedContacts $results)
     public function delNumbers(array $numbers, callable $onComplete)
     {
         $this->getUsersByPhones($numbers, function (array $contacts) use ($numbers, $onComplete) {
+            sleep(4);
```

Comment:
Is that really needed? Let's not leave such code in library, we expect it to be used in scenarios involving several simultaneous connections.